lib/miq_expression/tag.rb

```diff
@@ -19,11 +20,12 @@ def self.parse(field)
 
   def initialize(model, associations, column, managed = true)
     super(model, associations, column)
-    @namespace = "/#{managed ? MANAGED_NAMESPACE : USER_NAMESPACE}/#{column}"
+    @namespace = managed ? MANAGED_NAMESPACE : USER_NAMESPACE
+    @path = "/#{namespace}/#{column}"
```

Comment:
@imtayadeway what would be the proper name here? I chose path but...am not really sure (naming problems 😩 )